deepcell_label/labelmaker.py

```diff
@@ -63,55 +60,61 @@ def compute_feature_ids(self, feature):
         self.cell_ids[feature] = feature_cells
 
     def compute_info(self):
-        """
-        Make the cell_info dict.
-        """
+        """Create the cell_info dict."""
         self._cell_info = {}
         for feature in range(self.num_features):
             self.compute_feature_info(feature)
 
+    def compute_lineages(self):
+        """Make the cell_info dict."""
+        self._cell_info = {}
+        for feature in range(self.num_features):
+            self.compute_feature_lineage(feature)
+
     def compute_feature_info(self, feature):
         """
         Args:
             feature (int): which feature to recompute label metadata for
         """
         feature = int(feature)
         # Find the labels in the feature
-        feature_labels = self.labels[..., feature]
-        feature_cells = np.unique(feature_labels)[np.nonzero(np.unique(feature_labels))]
+        labels = self.labels[..., feature]
+        cells = np.unique(labels)[np.nonzero(np.unique(labels))]
         # Compute the label metadata for the feature
-        feature_info = {}
-        for cell in feature_cells:
+        info = {}
+        for cell in cells:
             cell = int(cell)
-            feature_info[cell] = {'label': str(cell),
-                                  'frames': [],
-                                  'slices': ''}
+            info[cell] = {'label': cell,
+                          'frames': [],
+                          'slices': ''}
             for frame in range(self.num_frames):
-                if cell in feature_labels[frame, ...]:
-                    feature_info[cell]['frames'].append(int(frame))
-        self.cell_ids[feature] = feature_cells
-        self.cell_info[feature] = feature_info
+                if cell in labels[frame, ...]:
+                    info[cell]['frames'].append(int(frame))
+        self.cell_ids[feature] = cells
+        self.cell_info[feature] = info
 
```

Comment:
I'm a little confused between `cell_info` and `_cell_info`. I think this is just setting the `_cell_info` dict directly? If I am understanding, can we just update to set `_cell_info` instead to be more clear?